packages/node_modules/@webex/plugin-meetings/src/config.js

```diff
@@ -89,7 +89,8 @@ export default {
     enableExtmap: false,
     experimental: {
       enableMediaNegotiatedEvent: false,
-      enableUnifiedMeetings: false
+      enableUnifiedMeetings: true,
```

Comment:
so we're enabling the use of wbxappapi for everyone by default as part of this PR too? shouldn't this be a separate PR?

Reply:
added for this PR for testing